src/editor/pages/flows/[id].tsx

```diff
@@ -81,30 +95,28 @@ const DataFlowOverview = ( { dataFlow }: any ): JSX.Element => {
       method: 'DELETE',
       headers: { 'Content-Type': 'application/json' },
       body: JSON.stringify( flow )
-    }  )
-    if( response.ok ) {
+    } )
+    if ( response.ok ) {
```

Comment:
Is this linted?

Reply:
yes, but my linter is still not working properly 😟